src/instructor/Questions.tsx

```diff
@@ -22,82 +24,105 @@ interface Props {
   questions: Question[];
   selectedQuestion: number | null;
   setSelectedQuestion: (value: number | null) => void;
-  onAddQuestion: (type: QuestionType) => void;
+  setQuestions: (questions: Question[]) => void;
 }
 
 export const Questions = ({
-  questions,
+  questions: qs,
```

Comment:
```suggestion
  questions: questionsFromProps,
```
Or another more descriptive label.